db/migrate/20191128111048_add_missing_foreign_keys.rb

```diff
@@ -0,0 +1,13 @@
+class AddMissingForeignKeys < ActiveRecord::Migration[5.2]
+  def change
+    add_foreign_key :api_secrets, :users, on_delete: :cascade
```

Comment:
`:cascade` means "delete all api secrets belonging to a user when the user is deleted"